File: 1483 - Tarea Geométrica I/problem_1483.java
import java.util.Scanner;


public class problem_1483 {
   public static void main(String[] args) {
        Scanner e = new Scanner(System.in);
        String cad = e.nextLine();
        String[] sep = cad.split(" ");
        int area;
        double areacir;
        if("rectangle".equals(sep[0]))
        {
            area = Integer.parseInt(sep[1]) * Integer.parseInt(sep[2]);
            System.out.println(area);
        }
        if("square".equals(sep[0]))
        {
            area = Integer.parseInt(sep[1]) * Integer.parseInt(sep[1]);
            System.out.println(area);
        }
        if("triangle".equals(sep[0]))
        {
            area = Integer.parseInt(sep[1]) * Integer.parseInt(sep[2]);
            System.out.println(area);
        }
        if("circle".equals(sep[0]))
        {
            areacir = Math.PI * (Double.parseDouble(sep[1]) * Double.parseDouble(sep[1]));
            System.out.println(areacir);
        }
        
    }
} 

File: 1483 - Tarea Geométrica I/problem_1483.cs
using System;

namespace problem_1483
{
    class Program
    {
        static void Main(string[] args)
        {
            String cad = Console.ReadLine();
            String[] sep = cad.Split(' ');
            int area;
            double areacir;
            if(sep[0].Equals("rectangle"))
            {
                area = Convert.ToInt32(sep[1]) * Convert.ToInt32(sep[2]);
                Console.WriteLine(area);
            }
            if(sep[0].Equals("square"))
            {
                area = Convert.ToInt32(sep[1]) * Convert.ToInt32(sep[1]);
                Console.WriteLine(area);
            }
            if(sep[0].Equals("triangle"))
            {
                area = Convert.ToInt32(sep[1]) * Convert.ToInt32(sep[2]);
                Console.WriteLine(area);
            }
            if(sep[0].Equals("circle"))
            {
                areacir = Math.PI * (Double.Parse(sep[1]) * Double.Parse(sep[1]));
                Console.WriteLine(areacir);
            }

        }
    }
}
